vault/resource_token_auth_backend_role_test.go

```diff
@@ -106,6 +108,8 @@ func TestAccTokenAuthBackendRoleUpdate(t *testing.T) {
 					resource.TestCheckResourceAttr("vault_token_auth_backend_role.role", "disallowed_policies_glob.#", "1"),
 					resource.TestCheckResourceAttr("vault_token_auth_backend_role.role", "disallowed_policies_glob.0", "def*"),
 					resource.TestCheckResourceAttr("vault_token_auth_backend_role.role", "orphan", "true"),
+					resource.TestCheckResourceAttr("vault_token_auth_backend_role.role", "allowed_entity_aliases.#", "1"),
+					resource.TestCheckResourceAttr("vault_token_auth_backend_role.role", "allowed_entity_aliases.1785148924", "test"),
```

Comment:
Better to get the value by its index rather than its hash:
```suggestion
					resource.TestCheckResourceAttr("vault_token_auth_backend_role.role", "allowed_entity_aliases.0", "test"),
```